test/Microsoft.ML.Tests/Microsoft.ML.Tests.csproj

```diff
@@ -33,6 +33,7 @@
   </ItemGroup>
   
   <ItemGroup>
+    <PackageReference Include="System.Memory" Version="4.5.1" />
```

Comment:
This shouldn't be necessary.  All our test projects are `netcoreapp2.1`, and so Span and ReadOnlyMemory should be available by default.